Fix:

error: redefinition of typedef ‘open_file_func’

https://github.com/libkml/libkml/issues/253

Part of
https://github.com/libkml/libkml/commit/d4adfd8a8862ce460314e514b26cffc9815f06ea
--- src/kml/base/contrib/minizip/iomem_simple.h.orig	2015-12-21 11:23:05.000000000 -0600
+++ src/kml/base/contrib/minizip/iomem_simple.h	2024-07-24 12:17:34.000000000 -0500
@@ -35,13 +35,6 @@
 extern "C" {
 #endif
 
-typedef voidpf (ZCALLBACK *open_file_func) OF((voidpf opaque, const char* filename, int mode));
-typedef uLong  (ZCALLBACK *read_file_func) OF((voidpf opaque, voidpf stream, void* buf, uLong size));
-typedef uLong  (ZCALLBACK *write_file_func) OF((voidpf opaque, voidpf stream, const void* buf, uLong size));
-typedef long   (ZCALLBACK *tell_file_func) OF((voidpf opaque, voidpf stream));
-typedef long   (ZCALLBACK *seek_file_func) OF((voidpf opaque, voidpf stream, uLong offset, int origin));
-typedef int    (ZCALLBACK *close_file_func) OF((voidpf opaque, voidpf stream));
-typedef int    (ZCALLBACK *testerror_file_func) OF((voidpf opaque, voidpf stream));
 /** KMR already defined in minizip/ioapi.h
 typedef struct zlib_filefunc_def_s
 {
